src/opf/sparse_sdpwrm.jl

```diff
@@ -379,4 +378,34 @@ function extract_dual(opf::OPFModel{SparseSDPOPF})
     end
 
     return dual_solution
-end
\ No newline at end of file
+end
+
+"""
+    _get_overlapping_pairs(groups)
+
+Get the indices of pairs of cliques in groups that overlap.
+
+Refer to https://github.com/lanl-ansi/PowerModels.jl/blob/be6af59202a6868b20a41214cb341b883d62e5f0/src/form/wrm.jl#L273-L274
+"""
+function _get_overlapping_pairs(groups::Vector{Vector{Int}})
+    # Compute the clique tree
+    tree = PM._prim(PM._overlap_graph(groups))
+    # Get the Cartesian indices of the adjacency matrix of the clique tree, which are exactly the indices of pairs
+    # of cliques that overlap
+    overlapping_pairs = [Tuple(CartesianIndices(tree)[i]) for i in (LinearIndices(tree))[findall(x->x!=0, tree)]]
+end
```

Comment:
Adding explicit return statement
```suggestion
    overlapping_pairs = [Tuple(CartesianIndices(tree)[i]) for i in (LinearIndices(tree))[findall(x->x!=0, tree)]]
    return overlapping_pairs
end
```